stubs/torch/__init__.pyi

```diff
@@ -28,6 +28,7 @@ from . import cuda as cuda
 from . import optim as optim
 from . import nn as nn
 from . import testing as testing
+from . import utils as utils
```

Comment:
I am including these two unrelated changed in a separate PR. If it goes in first, you can remove these. otherwise, merge them in, I will deal with conflicts.